app/api/public_v2.py

```diff
@@ -101,6 +101,16 @@ def __init__(self, daily_data):
             state = getattr(data_for_day, 'state') or 'US'   # state is 'US' if national
             self.key_to_date[state][data_for_day.date] = data_for_day
 
+    def population_percent(self, core_data, field_name):
+        field_value_for_day = getattr(core_data, field_name)
+        if field_value_for_day is None:
+            return None
+
+        state = getattr(core_data, 'state') or 'US'
+        state_population = population_lookup(state)
```

Comment:
small thing: it might be cleaner to use the property instead of the lookup function directory, so I'm guessing `core_data.state_obj.population` should work?

Reply:
I snuck a US population in there, so I called the lookup function directly so it would work when we add the US endpoints even though US isn't a state. Not sure if that's the bad kind of clever though.

Reply:
That seems okay, though maybe add a small comment here to that effect?